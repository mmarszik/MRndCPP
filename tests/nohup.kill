[1] 12343
[2] 12442
[3] 12531
[4] 12654

