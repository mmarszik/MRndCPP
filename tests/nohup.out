rm: cannot remove 'MRndCPP': No such file or directory
rm: cannot remove 'time.txt': No such file or directory
rm: cannot remove 'test.txt': No such file or directory
USE_RND_SIM_LIN_00
../use_rnd_sim_lin_00.h
USE_RND_SIM_LIN_01
../use_rnd_sim_lin_01.h
USE_RND_SIM_LIN_02
../use_rnd_sim_lin_02.h
USE_RND_SIM_LIN_04
../use_rnd_sim_lin_04.h
USE_RND_SIM_LIN_05
../use_rnd_sim_lin_05.h
USE_RND_SIM_LIN_06
../use_rnd_sim_lin_06.h
USE_RND_FIB_0
../use_rnd_fib_0.h
USE_RND_FIB_0a
../use_rnd_fib_0a.h
USE_RND_FIB_1
../use_rnd_fib_1.h
USE_RND_FIB_1a
../use_rnd_fib_1a.h
USE_RND_FIB_2
../use_rnd_fib_2.h
USE_RND_FIB_3
../use_rnd_fib_3.h
USE_RND_FIB_2a
../use_rnd_fib_2a.h
USE_RND_FIB_3a
../use_rnd_fib_3a.h
USE_RND_FIB_4
../use_rnd_fib_4.h
USE_RND_FIB_5
../use_rnd_fib_5.h
USE_RND_FIB_6
../use_rnd_fib_6.h
USE_RND_FIB_7
../use_rnd_fib_7.h
USE_RND_FIB_8
../use_rnd_fib_8.h
USE_RND_FIB_9
../use_rnd_fib_9.h
USE_RND_FIB_10
../use_rnd_fib_10.h
USE_RND_FIB_11
../use_rnd_fib_11.h
USE_RND_FIB_12
../use_rnd_fib_12.h
USE_RND_FIB_13
../use_rnd_fib_13.h
USE_RND_FIB_14
../use_rnd_fib_14.h
USE_RND_FIB_15
../use_rnd_fib_15.h
USE_RND_FIB_16
../use_rnd_fib_16.h
USE_RND_FIB_17
../use_rnd_fib_17.h
USE_RND_FIB_18
../use_rnd_fib_18.h
USE_RND_FIB_19
../use_rnd_fib_19.h
USE_RND_FIB_20
../use_rnd_fib_20.h
USE_RND_SFIB_0
../use_rnd_sfib_0.h
USE_RND_COMP_0
../use_rnd_comp_0.h
USE_RND_COMP_1
../use_rnd_comp_1.h
USE_RND_COMP_2
../use_rnd_comp_2.h
USE_RND_COMP_3
../use_rnd_comp_3.h
USE_RND_COMP_4
../use_rnd_comp_4.h
USE_RND_COMP_5
../use_rnd_comp_5.h
USE_RND_COMP_6
../use_rnd_comp_6.h
USE_RND_COMP_7
../use_rnd_comp_7.h
USE_RND_LIN_1
../use_rnd_lin_1.h
USE_RND_LIN_1a
../use_rnd_lin_1a.h
USE_RND_LIN_2
../use_rnd_lin_2.h
USE_RND_LIN_2a
../use_rnd_lin_2a.h
USE_RND_LIN_2b
../use_rnd_lin_2b.h
USE_RND_LIN_2c
../use_rnd_lin_2c.h
USE_RND_MLIN_0
../use_rnd_mlin_0.h
USE_RND_RANLUX_48
../use_rnd_ranlux_48.h
USE_RND_MT19937_64
../use_rnd_mt19937_64.h
USE_RND_KNUTH_B
../use_rnd_knuth_b.h
USE_RND_SKIP_LIN_1
../use_rnd_skip_lin_1.h
USE_RND_SKIP_LIN_2b
../use_rnd_skip_lin_2b.h
USE_RND_XOR2_0
../use_rnd_xor2_0.h
USE_RND_XOR3_0
../use_rnd_xor3_0.h
USE_RND_ADD_ROT_0a
../use_rnd_add_rot_0a.h
USE_RND_ADD_ROT_0b
../use_rnd_add_rot_0b.h
USE_RND_ADD_ROT_0c
../use_rnd_add_rot_0c.h
USE_RND_ADD_ROT_1a
../use_rnd_add_rot_1a.h
USE_RND_ADD_ROT_1b
../use_rnd_add_rot_1b.h
USE_RND_ADD_ROT_1c
../use_rnd_add_rot_1c.h
USE_RND_ADD_ROT_2a
../use_rnd_add_rot_2a.h
USE_RND_ADD_ROT_2b
../use_rnd_add_rot_2b.h
USE_RND_ADD_ROT_2c
../use_rnd_add_rot_2c.h
USE_RND_ADD_ROT_3a
../use_rnd_add_rot_3a.h
USE_RND_ADD_ROT_3b
../use_rnd_add_rot_3b.h
USE_RND_ADD_ROT_3c
../use_rnd_add_rot_3c.h
USE_RND_ADD_ROT_4a
../use_rnd_add_rot_4a.h
USE_RND_ADD_ROT_4b
../use_rnd_add_rot_4b.h
USE_RND_ADD_ROT_4c
../use_rnd_add_rot_4c.h
USE_RND_ADD_ROT_5a
../use_rnd_add_rot_5a.h
USE_RND_ADD_ROT_5b
../use_rnd_add_rot_5b.h
USE_RND_ADD_ROT_5c
../use_rnd_add_rot_5c.h
USE_RND_ADD_ROT_6a
../use_rnd_add_rot_6a.h
USE_RND_ADD_ROT_6b
../use_rnd_add_rot_6b.h
USE_RND_ADD_ROT_6c
../use_rnd_add_rot_6c.h
USE_RND_ADD_ROT_7a
../use_rnd_add_rot_7a.h
USE_RND_ADD_ROT_7b
../use_rnd_add_rot_7b.h
USE_RND_ADD_ROT_7c
../use_rnd_add_rot_7c.h
